modules/core/04-channel/keeper/packet.go

```diff
@@ -342,14 +342,12 @@ func (k Keeper) WriteAcknowledgement(
 		return types.ErrAcknowledgementExists
 	}
 
-	if len(acknowledgement) == 0 {
```

Comment:
How come the length check is removed? Should this not be kept?

Reply:
Yeah thanks, good catch. I was working under the assumption that an Acknowledgement wouldn't have empty ack bytes